packages/compass-aggregations/src/utils/stage.js

```diff
@@ -94,7 +96,7 @@ export function getStageOperator(stage) {
     return typeof ns === 'object' ? `${ns.db}.${ns.coll}` : `${database}.${ns}`;
   }
   if (stageOperator === '$out') {
-    if (stageValue.s3) {
+    if (stageValue.s3 || stageValue.atlas) {
```

Comment:
`$merge` doesn't need the same check? https://www.mongodb.com/docs/atlas/data-federation/supported-unsupported/pipeline/merge/#syntax

Reply:
Oh, good catch! Yeah, seems like it also needs this check, totally missed it